tests/test_cli.py

```diff
@@ -173,3 +177,28 @@ def test_create_build_image(self, temp_image_name):  # noqa
             self.exec_cli(command)
         assert excinfo.value.code == 0
         dt.remove_image(temp_image, noprune=True)
+
+    def test_log_encoding(self, caplog):
+        if MOCK:
+            mock_docker()
+
+        (flexmock(InputPluginsRunner)
+            .should_receive('run')
+            .and_raise(RuntimeError))
+
+        command = [
+            "main.py",
+            "--verbose",
+            "inside-build",
+        ]
+        with caplog.atLevel(logging.INFO):
+            with pytest.raises(RuntimeError):
+                self.exec_cli(command)
```

Comment:
Yeah, Tim added that last week and I pulled and pushed it out on Friday.  Those lines are easily seen in test_cli.py.  Did I miss something?